client/sttp-client4/src/main/scala/sttp/tapir/client/sttp4/basic/BasicSttpClientInterpreter.scala

```diff
@@ -0,0 +1,223 @@
+package sttp.tapir.client.sttp4.basic
+
+import sttp.client4.{Backend, Request}
+import sttp.model.Uri
+import sttp.tapir.client.sttp4.{SttpClientInterpreter, SttpClientOptions}
+import sttp.tapir.{DecodeResult, Endpoint, PublicEndpoint}
+
+trait BasicSttpClientInterpreter extends SttpClientInterpreter { // extends Extensions???
```

Comment:
I think I would avoid the `Basic...` nomenclature. Maybe we could simply call this `SttpClientInterpreter` and put it in the main package? + add in the scaladocs that for WS/streaming endpoints, please use XYZ instead

Reply:
done, I have removed the basic package altogether and renamed the BasicSttpXYZ to SttpXYZ